eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Providers/OperationMethodProviders/ResourceOperationMethodProvider.cs

```diff
@@ -200,7 +201,7 @@ private TryExpression BuildTryExpression()
             return new TryExpression(tryStatements);
         }
 
-        private IReadOnlyList<MethodBodyStatement> BuildClientPipelineProcessing(
+        protected virtual IReadOnlyList<MethodBodyStatement> BuildClientPipelineHandling(
```

Comment:
can we revert this method renaming? the original name seems better

Reply:
reverted